src/Layout/redist/targets/GenerateLayout.targets

```diff
@@ -2,7 +2,11 @@
 
   <Target Name="PublishPortableRuntimeIdentifierGraph"
           BeforeTargets="Build">
-    
+
+      <ItemGroup Condition="'$(DotNetBuildFromSource)' == 'true' AND '$(TargetRid)' != '' AND '$(TargetRid)' != '$(PortableRid)'">
```

Comment:
Should a `'$(PortableRid)' != ''` check be added as well?

Reply:
I figured it would be redundant, since `'$(TargetRid)' != '' AND '$(TargetRid)' != '$(PortableRid)'` would catch that case.

Reply:
If `$(PortableRid)` were empty then `'$(TargetRid)' != '' AND '$(TargetRid)' != '$(PortableRid)'` would evaluate to true as long as `$(TargetRid)` is not empty

Reply:
Oh wow, clearly my brain is not working.

Reply:
```suggestion
      <ItemGroup Condition="'$(DotNetBuildFromSource)' == 'true' AND '$(TargetRid)' != '' AND '$(PortableRid)' != '' AND '$(TargetRid)' != '$(PortableRid)'">
```

Reply:
I think we have all had those moments 😃